core/core-services-impl/src/main/java/com/sflpro/notifier/services/notification/impl/push/PushNotificationRecipientServiceImpl.java

```diff
@@ -72,6 +73,18 @@ private void assertSearchParameters(final PushNotificationRecipientSearchParamet
         Assert.notNull(searchParameters, "Search parameters should not be null");
     }
 
+    /*TODO chk with Mr. Smith*/
+    private PushNotificationRecipientSearchFilter convertParametersToFilter(PushNotificationRecipientSearchParameters parameters) {
+        PushNotificationRecipientSearchFilter filter = new PushNotificationRecipientSearchFilter();
+        filter.setApplicationType(parameters.getApplicationType());
+        filter.setDestinationRouteToken(parameters.getDestinationRouteToken());
+        filter.setDeviceOperatingSystemType(parameters.getDeviceOperatingSystemType());
+        filter.setProviderType(parameters.getProviderType());
+        filter.setStatus(parameters.getStatus());
+        filter.setSubscriptionId(parameters.getSubscriptionId());
+        return filter;
+    }
+
     /* Properties getters and setters */
     public PushNotificationRecipientRepository getPushNotificationRecipientRepository() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)